tests/python_functional/src/common/operations.py

```diff
@@ -50,3 +50,9 @@ def delete_session_file(shared_file_name, syslog_ng_testcase):
     working_dir = syslog_ng_testcase.testcase_parameters.get_working_dir()
     shared_file_name = Path(working_dir, shared_file_name)
     shared_file_name.unlink()
+
+
+def calculate_testcase_name(pytest_request):
+    # In case of parametrized tests we need to replace "[" and "]" because
+    # testcase name will appear in directory name
+    return pytest_request.node.name.replace("[", "_").replace("]", "_")
```

Comment:
The commit is a little misleading as you remove the `node.originalname` as well. Can we left that out ?

Reply:
Sorry, I forgot to mention it in the commit. So yes, we can leave originalname. It is the same as `node.name` but without any decorations for example: suffixes which came from parametrization, it is only the pure testcase function name. 
And we did not need this, because we always need the parametrization suffix to create separate directories also for parametrized tests. With using `node.name` we will get uniq testcase names also in case of parametrized tests.